config/pipelines.yml

```diff
@@ -237,15 +237,19 @@ test:
         name: *pcr_tag_set_name
         hostname: 'localhost:3000'
       message: *ont_covid_message
-
+        
 uat:
   <<: *default
   ont:
     covid:
       pcr_tag_set:
         name: *pcr_tag_set_name
```

Comment:
This references the value in the main block above, but it probably shouldn't be there.  Maybe this should be defined for the first time in the `development:` settings and then referenced for the other 3 environments from there.